src/app/modules/item/pages/item-forum/item-forum.component.html

```diff
@@ -0,0 +1,95 @@
+<ng-container *ngrxLet="selected$; let selected">
+  <h2 class="title alg-h2">Help requests</h2>
+  <alg-selection
+    [items]="options"
+    [selected]="selected"
+    (change)="onChange($event, options)"
+    *ngIf="options$ | async as options"
+  ></alg-selection>
+
+  <alg-error
+    class="alg-flex-1"
+    i18n-message message="You are not currently observing any group."
+    *ngIf="selected === 2 && !(isWatching$ | async); else contentSection"
+  ></alg-error>
+
+  <ng-template #contentSection>
+    <ng-container *ngIf="state$ | async as state">
+      <alg-loading class="alg-flex-1" size="medium" *ngIf="state.isFetching && !state.data"></alg-loading>
+      <alg-error
+        *ngIf="state.isError"
+        class="alg-flex-1"
+        icon="fa fa-exclamation-triangle"
+        i18n-message message="Unable to load the recent threads"
+        i18n-buttonCaption buttonCaption="Retry"
+        [showRefreshButton]="true"
+        (refresh)="refresh()"
+      ></alg-error>
+
+      <ng-container *ngIf="state.data">
+        <div class="operation-buttons">
+          <button class="alg-button basic" pButton icon="ph-duotone ph-arrows-clockwise" (click)="refresh()">
+            Refresh
+          </button>
+        </div>
+        <p-table
+          class="alg-table"
+          [columns]="state.data.columns"
```

Comment:
As the goal is not (anymore) to play with the column, reorder them, dynamically remove/add some... I would prefer we just use explicit `<td>` instead of iterating over columns which make it more difficult to read (in particular with the switch/case) and less type-safe. 
Don't change it this time of course, but just keep it in mind for the future.